.github/workflows/patch-release-build.yml

```diff
@@ -99,6 +99,67 @@ jobs:
           GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
 
+      - name: Create checksum
+        run: |
+          echo ORIG_CHECKSUM="$(shasum otelagent/build/libs/aws-opentelemetry-agent-*.jar | awk '{ print $1 }')" >> "$GITHUB_ENV"
+
+      - name: Get current version
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        shell: bash
+        run: |
+          echo ADOT_JAVA_VERSION="$(./gradlew printVersion -q )" >> $GITHUB_ENV
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+
+      - name: Build image
+        uses: docker/build-push-action@v4
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        with:
+          push: false
+          build-args: "ADOT_JAVA_VERSION=${{ env.ADOT_JAVA_VERSION }}"
+          context: .
+          platforms: linux/amd64,linux/arm64
+          tags: ${{ env.TEST_TAG }}
+
+      - name: Test docker image
+        shell: bash
+        run: |
+          docker volume create operator-volume
```

Comment:
can we transform this into a script and host it here? https://github.com/aws-observability/aws-otel-java-instrumentation/tree/main/.github/scripts

This will be used in several workflows, so it makes sense to reuse it. `TEST_TAG` can be passed as parameter to the script.